common/static/common/js/discussion/utils.js

```diff
@@ -386,8 +386,7 @@
                         /*
                          bug fix, ordering is off
```

Comment:
As discussed in this [ticket](https://openedx.atlassian.net/browse/TNL-5451) Chris Dodge has mentioned that it was a conflict resolved in this [commit](https://github.com/edx/edx-platform/commit/7229a9cdaf459d6d99a63ebed843e8680fd7c7e7), at that time this comment was being added. This comment and code followed by it replaced the exact code I wrote to fix the bug described in [ticket](https://openedx.atlassian.net/browse/TNL-5451). Before this [commit](https://github.com/edx/edx-platform/commit/7229a9cdaf459d6d99a63ebed843e8680fd7c7e7) code was same as I have now in this branch for that specific segment(responsible for ordering). Till now I am unable to find why ordering was intentionally set to off. Please guide me how I can dug further to get reason behind this.
Thank You.